docs/about/compatibility/precission-support.md

```diff
@@ -0,0 +1,128 @@
+| float8 (E5M2)   | -             | A 8-bit floating point number mostly following IEEE-754 conventions and bit layout **S1E5M2** described in [8-bit Numerical Formats for Deep Neural Networks](https://arxiv.org/abs/2206.02915), with expanded range and with no infinity or signed zero. NaN is represented as negative zero. |
+| float16         | ``half``     | A 16-bit floating-point number, which is conform to the IEEE 754-2008 single precision storage format. |
+| bfloat16        | ``bfloat16`` | A 16-bit floating-point number, which is shortened version of IEEE 754 single precision storage format. |
+| tensorfloat32   | -            | Floating point number that occupies 32 bits or less of storage, providing improved range compared to half (16-bit) formats, at (potentially) greater throughput than single precision (32-bit) formats. |
+| float32         | ``float``    | A 32-bit floating-point number, which is conform to the IEEE 754 single precision storage format. |
+| float64         | ``double``   | A 64-bit floating-point number, which is conform to the IEEE 754 double precision storage format. |
+
+Notes:
+
+* At float8 types and tensorfloat32 type are internal types in Matrix Core and can be stored in any types.
+* The encodings for FP8 (E5M2) and FP8 (E4M3) natively supported by MI300 differ from the FP8 (E5M2) and FP8 (E4M3) encodings used in H100 ([FP8 FORMAT SFOR DEEP LEARNING](https://arxiv.org/abs/2209.05433)).
+* float8 (E5M2) at some AMD documents and articles named as bfloat8.
+
+![Supported floating point data types)](../../data/about/compatibility/floating-point-data-types.png "Supported floating point data types")
+
+# The support icon description
+
+| The support icon explanation                      | Support icon  |
+|:--------------------------------------------------|:-------------:|
+| Not supporting the specific type                  | ❌           |
+| Partial support of the specific type              | ⚠️           |
+| Support the specific type without HW acceleration | ✔            |
+| Native support of the specific type               | ✅           |
+
+# Native hardware support
+
+## Hardware accelerated compute units support
+
+| Data types   | int8  | int16  | int32  | int64  | float8 (E4M3) | float8 (E5M2) | float16 | bfloat16 | tensorfloat32 | float32 | float64 |
+|:-------------|:-----:|:------:|:------:|:------:|:-------------:|:-------------:|:-------:|:--------:|:-------------:|:-------:|:-------:|
+| Mi100        | ✅    |  ✅    | ✅    | ✅    | ❌           | ❌           | ✅      | ✅       | ❌           | ✅      | ✅     |
+| Mi200 series | ✅    |  ✅    | ✅    | ✅    | ❌           | ❌           | ✅      | ✅       | ❌           | ✅      | ✅     |
+| Mi300 series | ✅    |  ✅    | ✅    | ✅    | ❌           | ❌           | ✅      | ✅       | ❌           | ✅      | ✅     |
```

Comment:
Please check these again. Afaik `int64` is emulated and therefore runs at a lower rate than `int32`. Also the MI100 has half the float64 throughput as float32, but I wouldn't say, that it doesn't have "hardware acceleration"

Reply:
Let's talk.